SU-HSDO/suhumsci
# STN-199 polish integration

# [STN-199](https://sparkbox.atlassian.net/browse/STN-199) READY FOR REVIEW

## Summary
This PR includes a portion of design refinements documented in [STN-199](https://sparkbox.atlassian.net/browse/STN-199).


Designer: @meranicosme 
Developer: @heatherdesigns 

## Need Review By (Date)
EOD 3/16/2020

## Urgency
high

## Steps to Test
1. In the CLI, run `npm run test` to verify all tests pass.
2. Go to http://economics.suhumsci.loc. Verify the following design refinements have been implemented:
 - [x] Darken base secondary mint color to: #148762. Passes color contrast
 - [x] Make sure people component has a border bottom: Driftwood color: #B6B1A9
 - [x] Main Navigation Dropdown links should be driftwood and then on hover change to black
 - [x] Main Navigation: Links should be font-weight: semibold
 - [x] Hero 1: Check the vertical size for mobile. Looks a bit too long but double check (height was adjusted0
 - [x] Speed up navigation arrow animation on desktop
 - [x] Add drop shadow to desktop nav dropdown and change to driftwood and 1px width
 - [x] Desktop navigation border bottom should be driftwood and 1px

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
